documentation/acceptence-tests/budget-management-tests.md

```diff
@@ -0,0 +1,18 @@
+# Budget Management Acceptance Tests
+
+## Acceptance test summary
+
+### create, view and modify budget
+
+1. Login as valid user with expenses
+2. Navigate to the Budgets section
```

Comment:
Can you be more descriptive like saying 'navigate to the budgets section by clicking on the home page 'see all your budgets' jsut fo the ones that are a lil vague